.github/workflows/bootstrap.yml

```
@@ -0,0 +1,76 @@
+# this workflow bootstraps the testing of the build the docker images
+# 
+# - this will run the python script used to generate the workflows 
+#   based on a the jinja template
+# - commit and push the generated workflows to the branch test_docker_build 
+#   where they will be executed
+
+name: bootstrap
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches: [ "test_build" ] 
```

```suggestion
    branches: [ "master" ] 
```

to be changed before merging